components/Leaderboard/style.module.css

```diff
@@ -68,28 +68,78 @@
     rgba(150, 93, 233, 1) 10.8%,
     rgba(99, 88, 238, 1) 94.3%
   );
-  border-radius: 500px;
+  border-radius: 30px;
   height: 100%;
   min-width: 55px;
 }
 
 .bar_first {
   background: linear-gradient(225deg, #fbf794 0%, #c67700 100%);
-  border-radius: 500px;
+  border-radius: 30px;
   height: 100%;
   min-width: 55px;
 }
 
 .bar_second {
   background: linear-gradient(45deg, #6c7175, #eee);
-  border-radius: 500px;
+  border-radius: 30px;
   height: 100%;
   min-width: 55px;
 }
 
 .bar_third {
   background-image: linear-gradient(45deg, #826055 0%, #dfa579 100%);
-  border-radius: 500px;
+  border-radius: 30px;
   height: 100%;
   min-width: 55px;
 }
+
+.sort {
+  padding-bottom: 20px;
+  text-align: center;
+  border-bottom: 1px solid #eee;
+  font-family: 'Noto Sans', sans-serif;
+  font-weight: 500;
+  font-size: 14px;
+  color: #404040;
+}
+
+.sort_back {
+  background-color: white;
+  border: 5px solid white;
+  border-radius: 50px;
+  box-shadow: rgba(0, 0, 0, 0.2) 0px 1px 10px;
+  width: fit-content;
+  margin: auto;
+}
+
+.sort_button {
+  background: rgba(255, 118, 73, 0.9);
+  border-radius: 999px;
+  box-sizing: border-box;
+  color: #ffffff;
+  cursor: pointer;
+  font-family: Inter, Helvetica, 'Apple Color Emoji', 'Segoe UI Emoji',
+    NotoColorEmoji, 'Noto Color Emoji', 'Segoe UI Symbol', 'Android Emoji',
+    EmojiSymbols, -apple-system, system-ui, 'Segoe UI', Roboto, 'Helvetica Neue',
+    'Noto Sans', sans-serif;
+  font-size: 13px;
+  font-weight: 500;
+  line-height: 24px;
+  opacity: 1;
+  outline: 0 solid transparent;
+  padding: 8px 16px;
+  user-select: none;
+  -webkit-user-select: none;
+  touch-action: manipulation;
+  width: fit-content;
+  word-break: break-word;
+  border: 0;
+  margin: 0 3px 0 3px;
+}
+
+.leading_info {
```

Comment:
We should probably write our CSS module classes in PascalCase in the future :bulb:

Reply:
Noted, will make sure to do a PR in the future changing all CSS to PascalCase :muscle:

Reply:
That would be awesome 🚀